proto/mls/database/intents.proto

```diff
@@ -62,9 +63,14 @@ message RemoveMembersData {
 
 // Generic data-type for all post-commit actions
 message PostCommitAction {
+  // An installation
+  message Installation {
+    bytes installation_id = 1;
```

Comment:
Want to rename this to `installation_key` to be consistent with the rename we did in https://github.com/xmtp/proto/pull/127?